static/main.js

```diff
@@ -0,0 +1,28 @@
+const notificationForm = document.getElementById('notificationForm')
+notificationForm.addEventListener('submit', function(event){
+    event.preventDefault();
+    createNotification();
+});
+
+function createNotification(){
+    const form = notificationForm
+    let title = document.getElementById('title').value
+    let description = document.getElementById('description').value
+
+    fetch('http://localhost:8000/notifications', {
```

Comment:
This ideally should not remain hardcoded, but I kept it like this for now to ease local development. Will be fetching it from env vars. Need to figure out that how to use environment setup for a frontend standalone webpage.

Reply:
This also looks good considering it will be a considerable amount of effort.